.gitlab-ci.yml

```diff
@@ -77,7 +77,7 @@ test_acceptance_prebuilt_raspberrypi:
     - tar xzf deploy.tar.gz
     # Extract converted Raspbian artifacts
     - unxz deploy/raspberrypi-${RASPBIAN_NAME}-mender-${MENDER_CLIENT_VERSION}-mender.img.xz
```

Comment:
There is still missing the update on the `unxz` command.

Reply:
Doh! Thanks, updated!